tools/appliance/builtin/template.json

```diff
@@ -23,7 +23,7 @@
       "disk_interface": "virtio",
       "net_device": "virtio-net",
 
-      "iso_url": "http://mirror.nbrc.ac.in/centos/7/isos/x86_64/CentOS-7-x86_64-Minimal-1808.iso",
+      "iso_url": "http://mirror.nbrc.ac.in/centos/7/isos/x86_64/CentOS-7-x86_64-Minimal-1804.iso",
```

Comment:
Is this hash correct?

I took the hash of that file and it is something else:
```
$ wget http://mirror.nbrc.ac.in/centos/7/isos/x86_64/CentOS-7-x86_64-Minimal-1804.iso
$ md5sum CentOS-7-x86_64-Minimal-1804.iso
fabdc67ff3a1674a489953effa285dfd  CentOS-7-x86_64-Minimal-1804.iso
```

Reply:
Actually I was just about to send the exact comment. The official has only `sha1` and `sha256` hashes (which is used in `systemvmtemplate/template.json`) but here the `iso_checksum_type` is set to `md5`. Either of those needs to be changed.

Reply:
if the type can be set to sha1 or sha256, please do so. Otherwise, we need to configure the right hash value there.